src/coreclr/jit/gentree.cpp

```diff
@@ -21017,6 +21017,20 @@ GenTree* Compiler::gtNewSimdBinOpNode(
 
             return gtNewSimdBinOpNode(GT_AND, type, shiftOp, maskOp, simdBaseJitType, simdSize);
         }
+
+        case GT_LSH:
+        {
+            assert(varTypeIsByte(simdBaseType));
+            GenTree* andShiftVal = fgMakeMultiUse(&op2ForLookup);
+            GenTree* shiftOp =
+                gtNewSimdBinOpNode(GT_LSH, type, op1, op2ForLookup,
+                                   varTypeIsUnsigned(simdBaseType) ? CORINFO_TYPE_USHORT : CORINFO_TYPE_SHORT,
+                                   simdSize);
+            GenTree* maskElement =
+                gtNewOperNode(GT_LSH, andShiftVal->TypeGet(), gtNewIconNode(0xFF), fgMakeMultiUse(&andShiftVal));
```

Comment:
This can just be `andShiftVal`, you've already made it multi-use on L21024 with the first use in `shiftOp` and the second use here. There isn't a third use, so we don't need to do it again.